test/e2e/common/apparmor.go

```diff
@@ -61,12 +61,24 @@ if touch %[1]s; then
 elif ! touch %[2]s; then
   echo "FAILURE: write to %[2]s should be allowed"
   exit 2
-elif ! grep "%[3]s" /proc/self/attr/current; then
+elif ! fgrep -q "%[3]s" /proc/self/attr/current; then
```

Comment:
nit: the [man pages](https://linux.die.net/man/1/fgrep) say that `fgrep` is deprecated, in favor of `grep -F`.

Reply:
Actually, we want to match the full file contents, not just a substring, so I think a better check would be:
```shell
elif [[ $(< /proc/self/attr/current) != "%[3]s" ]]; then
  ...
```
Same below.